.github/workflows/windows.yml

```
@@ -32,27 +32,40 @@ jobs:
           pathInArchive: "binaryen-/bin/wasm-opt.exe"
 
       - name: Checkout sources & submodules
-        uses: actions/checkout@master
+        uses: actions/checkout@v2.4.0
```

```suggestion
        uses: actions/checkout@v2
```
It is recommended to use this tag as it following updates to the latest minor version